src/pages/landing/forex-trading/components/_hero.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import { Flex, Container, Show } from 'components/containers'
+import { Header } from 'components/elements'
+import { localize } from 'components/localization'
+import { Background } from 'components/elements/background-image'
+import { LinkButton } from 'components/form'
+import device from 'themes/device.js'
+
+const BackgroundWrapper = styled(Background)`
+    background-size: cover;
+    background-position: bottom right;
+`
+
+const Wrapper = styled(Container)`
+    @media ${device.tabletS} {
+        height: 70rem;
+        margin-left: 0;
+        padding: 2rem 16px 0;
+        flex-direction: column-reverse;
+        justify-content: center;
+    }
+`
```

Comment:
Pleaase pass these directly to the `tabletS` prop. And please do so whenever possible for the supported properties for components like `Flex`, `Box`, `Container`, `Header` & `Text`.